lxd/instance/instancetype/instance.go

```diff
@@ -735,6 +735,16 @@ var InstanceConfigKeysContainer = map[string]func(value string) error{
 	//  shortdesc: Whether to protect the file system from being UID/GID shifted
 	"security.protection.shift": validate.Optional(validate.IsBool),
 
+	// lxdmeta:generate(entities=instance; group=security; key=security.protection.start)
```

Comment:
As this should be a setting for both VMs and containers, use the `var InstanceConfigKeysAny` instead.
Also the `shortdesc` still has the text from the `security.protection.shift` key.